action.yml

```diff
@@ -92,8 +92,7 @@ runs:
       run: |
         pyinstaller \
           --clean \
-          --noconfirm \
```

Comment:
It is my personal preference to let it use `--noconfirm` by default. 

Although I would allow if you could add some way to bypass it through some input and let it use `--noconfirm` by default.

Reply:
I'll add it right away

Reply:
Added !